src/main/java/pl/javastart/task/Main.java

```diff
@@ -4,5 +4,14 @@ public class Main {
 
     public static void main(String[] args) {
 
+        String name1 = "Mleko";
+        String name2 = "Czekolada";
+        String producer1 = "Mlekowita";
+        String producer2 = "Wedel";
+        double price1 = 2.5;
+        double price2 = 2.19;
+
+        System.out.printf("%s %s %.1fzł%n", name1, producer1, price1);
```

Comment:
Przy walucie zawsze lepiej używać %.2f, jeżeli cena się zmieni np. na 2.54, to chcemy widzieć te 4 grosze na końcu, a wyświetlanie 2,50 zamiast 2,5 też jest ok